tests/lib/views/login/view.js

```diff
@@ -14,8 +14,13 @@ LoginView.prototype = Object.assign({
       this.accessors.passwordField.sendKeys(password);
       this.accessors.submitButton.click();
     }).then(() => {
-      return new require('../services/view')(this.driver);
+      var ServicesView = require('../services/view');
+      return new ServicesView(this.driver);
     });
+  },
+
+  isLoginButton: function() {
```

Comment:
Views shouldn't expose HtmlElements. How about moving `this.accessors.startLoginButton` in the constructor of LoginView? Then when when `LoginView` [is created here](https://github.com/fxbox/app/pull/17/files#diff-da27a7816cd34a151cc34d117ce6cf7eR17), we're sure the login button is visible.